src/Tribe/Cost_Utils.php

```diff
@@ -279,11 +288,13 @@ protected function get_cost_by_func( $costs = null, $function = 'max' ) {
 	 * If a range isn't provided, the resulting array will hold a single
 	 * value.
 	 *
-	 * @param $cost string Cost to parse.
+	 * @param string|array $costs        A cost string or an array of cost strings.
+	 * @param null         $max_decimals The maximum number of decimal values that should be returned in the range.
+	 * @param bool         $sort         Whether the returned values should be sorted.
 	 *
-	 * @return array
+	 * @return array An associative array of parsed costs in [ <string cost> => <cost number> ] format.
 	 */
-	public function parse_cost_range( $costs, $max_decimals = null ) {
+	public function parse_cost_range( $costs, $max_decimals = null, $sort = false ) {
```

Comment:
It really is a mistake, the param was previously named `do_not_sort` and the default stuck, good catch.